Packages/io.chainsafe.web3-unity.hyperplay/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "io.chainsafe.web3-unity.hyperplay",
-    "version": "1.0.0",
+    "version": "2.6.0",
```

Comment:
This should remain the same, I did not mention any version changes, just rectifying version mismatch between packages and samples.

Reply:
![image](https://github.com/ChainSafe/web3.unity/assets/57473220/60628e2e-fb0e-4508-af9d-b8cac15fee81)

?

Reply:
just tell me what versions you want each of the packages to be and ill amend the PR :)

Reply:
lets get real clear cut with this so we don't misunderstand each other

when i say sample folder version i mean these
![image](https://github.com/ChainSafe/web3.unity/assets/57473220/46150562-eeb6-4501-abc9-0c6320e134b2)

when i say json package version i mean these
![image](https://github.com/ChainSafe/web3.unity/assets/57473220/ba5f84c2-f7ce-4a25-b51f-2c0b8e333c16)

currently in this PR they are:

SDK
sample folder version: 2.6
package json version: 2.6

Hyperplay
sample folder version: 2.6
package json version: 2.6

Web3Auth
sample folder version: 2.6
package json version: 2.6

Lootboxes
sample folder version: 2.6
package json version: 2.6

Ramp
sample folder version: 2.6
package json version: 2.6

Marketplace
sample folder version: 2.6
package json version: 2.6

Please copy the above and replace the values to what you would like.
@kantagara can you weigh in on this too please.

Reply:
@sneakzttv all I requested was for version mismatch between samples and packages to be rectified, the easier way to do it is to delete samples and reimport samples, that is all. If it's still not clear I can take over, no problem